api/package.json

```diff
@@ -18,7 +18,8 @@
         "jobs": "ts-node jobs/index.ts",
         "buildprocessors": "node_modules/.bin/tsc jobs/processors/*.ts --outDir jobs/processors/build/",
         "buildprocessorsandrunjobs": "npm run buildprocessors && npm run jobs",
-        "dev-jobs": "nodemon --watch ./ -e ts --exec npm run buildprocessorsandrunjobs"
+        "dev-jobs": "nodemon --watch ./ -e ts --exec npm run buildprocessorsandrunjobs",
+        "typeorm:cli": "ts-node ./node_modules/typeorm/cli.js -f ./models/ormconfig.ts"
```

Comment:
What's this do? Is it used somewhere else?

Reply:
I think this will be useful if we encounter any QA or Production errors. Typeorm has difficult time finding typescript ormconfig files, so the defaul `typeorm do stuff` functionality wouldn't work without the file path `-f . . .`. This seemed like an easier way to get into the api container and use it if necessary. Also found it useful for local development.